pennylane_forest/converter.py

```diff
@@ -324,7 +324,7 @@ class ProgramLoader:
         program (pyquil.quil.Program): The pyquil Program instance that should be loaded
     """
 
-    _matrix_dictionary = pyquil.gate_matrices.QUANTUM_GATES
+    _matrix_dictionary = pyquil.simulation.matrices.QUANTUM_GATES
```

Comment:
Would suggest bumping to `pyquil>=2.7` in `requirements.txt`

Reply:
Ah, good catch!